### PR TITLE
Inital commit to add TensorFlow processor #84

### PR DESCRIPTION
**Pull request by tzolov**
Current TensorFlow SCDF Processor project is at https://github.com/tzolov/tensorflow-spring-cloud-stream-app-starters

I have added only the core `tensorflow` processor not the `twitter-sentiment processor

Resolves https://github.com/spring-cloud-stream-app-starters/app-starters-release/issues/84.

### CODE REVIEW

**Comment by sabbyanandan** [comment]
Thanks, @tzolov!

Hi, @artembilan @sobychacko @ericbottard - please review this at your convenience. The goal for this PR is to gather feedback and whenever it is ready, I'd like for it to be contributed to its own repo. Possibly `spring-cloud-stream-app-starters/tensorflow` - TBD.

**Comment by sabbyanandan** [comment]
Actually, this commit includes only the static descriptor files. The actual project will be submitted as a PR in https://github.com/spring-cloud-stream-app-starters/tensorflow.

**Comment by tzolov** [comment]
Because the  https://github.com/spring-cloud-stream-app-starters/tensorflow is empyt Github doesn't allow me to fork it. There is no button or workaround. 
Perhaps you can add an empty README to the project.

**Comment by jvalkeal** [comment]
@tzolov just added empty readme, repo is now forkable.

**Comment by artembilan** [comment]
This PR does not make sense without any original project.
Should be mark "On hold" or `[DON NOT MERGE YET]`

@tzolov , 
please, consider to stop PR'ing from your `master`. 
It is pretty easy to perform `git checkout -b TensorFlowProcessor` to create a new branch and switch to it directly.
That way it would be easier for us to review and merge on the our local side.
And after merging you will need only `git pull upstream master` to your local copy.
Right now you will have to do `git reset --hard upstream/master` with some other possible cleaning and resolving conflicts work.

**Comment by sabbyanandan** [comment]
@artembilan: It was my fault - I suggested submitting a PR without a new repo - sorry for the confusion. @tzolov is going to drop the remaining part of PR in https://github.com/spring-cloud-stream-app-starters/tensorflow.

**Comment by artembilan** [comment]
@sabbyanandan ,
thank you for taking responsibility, but that isn't an error or a fault at all, really!
I just suggest to mark the PR somehow to let other to know do not merge it by accident.

The way you suggested (this PR) we have finally that repo and it is ready for forking and pushing.
Otherwise I don't know how things would go.

E.g. I asked once about repo for the AWS Kinesis Binder, but nothing happened 😉

**Comment by tzolov** [comment]
here is the PR: https://github.com/spring-cloud-stream-app-starters/tensorflow/pull/1

**Comment by tzolov** [comment]
also should we discuss the review (and eventual rework) here or on the tensorflow#PR itself?

**Comment by artembilan** [comment]
Thank you for update, @tzolov !
Well, we will discuss here and there, but already in the scope of particular changes.

**Comment by artembilan** [comment]
I've marked this one with the `[DO NOT MEGE YET]`.
I'm fully sure that we should merge this one until that in `tensorflow`

**Comment by tzolov** [comment]
@artembilan i see and completely relate with your point.  This PR shouldn't be merged until the tensorflow processor PR get resolved.

File: pom.xml
Comment (artembilan) [inline pom.xml:67]: I'm not sure in the purpose of this project, but I guess the version here must be `1.0.0.BUILD-SNAPSHOT`

**Review by artembilan — CHANGES_REQUESTED** [review]

**Comment by sobychacko** [comment]
Updated the tensorflow processor version. 
LGTM 
@tzolov you covered pretty much everything needed for the docs and other metadata generation. Great work!
Here is the CI plan for the Tensorflow app starter: https://jenkins.spring.io/view/Seeds/job/spring-scst-app-starters-tensorflow-master-ci/

Merging.

**Comment by tzolov** [comment]
Great! Tanks guys!
Currently the tensorflow project contains two processors: the core `tensorflow-processor` and `twitter-sentiment-processor`, but this PR covers only the `tensorflow-processor` i believe? 
Would the `twitter-sentiment-processor` be picked somehow automatically (doesn't seem so) or
should i open another PR for the `twitter-sentiment-processor`? 
@sabbyanandan @sobychacko @artembilan WDYT?

**Pull request closed without merging**
